cgen/__init__.py

```diff
@@ -295,6 +295,18 @@ def get_decl_pair(self):
     mapper_method = "map_aligned"
 
 
+class AlignValueAttribute(NestedDeclarator):
```

Comment:
Add a references/context:

- https://www.intel.com/content/www/us/en/develop/documentation/oneapi-dpcpp-cpp-compiler-dev-guide-and-reference/top/compiler-reference/attributes/align-value.html
- https://reviews.llvm.org/D4635

maybe also explaining the difference with the "normal" `aligned` attribute. (which I'm still not sure I understand) Reading the [GCC docs](https://gcc.gnu.org/onlinedocs/gcc/C-Extensions.html#C-Extensions) suggests that the attribute applies to the whole variable no matter where in the declarator it occurs, though I'm not totally sure about that.

Reply:
Done in 1ddbb6752fccb1bb1335f9c08be44078fb223c1d

Reply:
Nice, thanks!

Reply:
Oh, almost forgot: please hook this into the docs.